[vulkan] `make reftests` failing (panicked at 'attempt to shift right with overflow')

Short info header:
- GFX version: `fdfe887f5afab7ca057e3e824817d0bcdcf794ba` on `master` (post 0.5.0)
- OS: Ubuntu 19.04 (`nvidia-driver-418`)
- GPU: GeForce GTX 970/PCIe/SSE2

It seems to fail for various scenes (transfer, compute, vertex-offset, basic .... on each run a randomly different scene fails) with always the same issue (` 'attempt to shift right with overflow'`), e.g.:

```
Parsing test suite 'local'...
Testing Vulkan:
	Scene 'compute':
thread 'main' panicked at 'attempt to shift right with overflow', src/backend/vulkan/src/command.rs:36:15
stack backtrace:
   0: backtrace::backtrace::libunwind::trace
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/backtrace/libunwind.rs:88
   1: backtrace::backtrace::trace_unsynchronized
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/backtrace/mod.rs:66
   2: std::sys_common::backtrace::_print_fmt
             at src/libstd/sys_common/backtrace.rs:77
   3: <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt
             at src/libstd/sys_common/backtrace.rs:59
   4: core::fmt::write
             at src/libcore/fmt/mod.rs:1052
   5: std::io::Write::write_fmt
             at src/libstd/io/mod.rs:1426
   6: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:62
   7: std::sys_common::backtrace::print
             at src/libstd/sys_common/backtrace.rs:49
   8: std::panicking::default_hook::{{closure}}
             at src/libstd/panicking.rs:204
   9: std::panicking::default_hook
             at src/libstd/panicking.rs:224
  10: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:472
  11: rust_begin_unwind
             at src/libstd/panicking.rs:380
  12: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
  13: core::panicking::panic
             at src/libcore/panicking.rs:52
  14: gfx_backend_vulkan::command::debug_color
             at src/backend/vulkan/src/command.rs:36
  15: <gfx_backend_vulkan::command::CommandBuffer as gfx_hal::command::CommandBuffer<gfx_backend_vulkan::Backend>>::insert_debug_marker
             at src/backend/vulkan/src/command.rs:984
  16: gfx_warden::gpu::Scene<B>::new
             at ./src/gpu.rs:286
  17: reftest::Harness::run_instance
             at src/warden/src/bin/reftest.rs:150
  18: reftest::Harness::run
             at src/warden/src/bin/reftest.rs:107
  19: reftest::main
             at src/warden/src/bin/reftest.rs:224
  20: std::rt::lang_start::{{closure}}
             at /rustc/b8cedc00407a4c56a3bda1ed605c6fc166655447/src/libstd/rt.rs:67
  21: std::rt::lang_start_internal::{{closure}}
             at src/libstd/rt.rs:52
  22: std::panicking::try::do_call
             at src/libstd/panicking.rs:305
  23: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:86
  24: std::panicking::try
             at src/libstd/panicking.rs:281
  25: std::panic::catch_unwind
             at src/libstd/panic.rs:394
  26: std::rt::lang_start_internal
             at src/libstd/rt.rs:51
  27: std::rt::lang_start
             at /rustc/b8cedc00407a4c56a3bda1ed605c6fc166655447/src/libstd/rt.rs:67
  28: main
  29: __libc_start_main
  30: _start
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
make: *** [Makefile:66: reftests] Error 101
```

--- a/src/backend/vulkan/src/command.rs
+++ b/src/backend/vulkan/src/command.rs
@@ -33,7 +33,7 @@ pub struct CommandBuffer {
 fn debug_color(color: u32) -> [f32; 4] {
     let mut result = [0.0; 4];
     for (i, c) in result.iter_mut().enumerate() {
-        *c = ((color >> (24 + i * 8)) & 0xFF) as f32 / 255.0;
+        *c = ((color >> (24 - i * 8)) & 0xFF) as f32 / 255.0;
     }
     result
 }

